packages/@okta/vuepress-site/docs/guides/ie-limitations/index.md

```diff
@@ -125,6 +125,22 @@ In the Admin Console, the enablement of a Registration Inline Hook has changed f
 
 ***
 
+#### The ‘audience’ parameter in the Authentication API
+
+**What Changed:** Passing the `audience` parameter to the `/api/v1/authn` API isn't supported in Identity Engine because of the new flexible App Sign-On Policy that comes with Identity Engine. The Classic Engine pipeline doesn't support the flexible App Sign-On Policy.
```

Comment:
```suggestion
**What Changed:** Passing the `audience` parameter to the `/api/v1/authn` API isn't supported in Identity Engine because of the new flexible App Sign-On Policy that comes with the Identity Engine. The Classic Engine pipeline doesn't support the flexible App Sign-On Policy.
```